src/test/sigencoding_tests.cpp

```diff
@@ -48,8 +58,9 @@ static void CheckSignatureEncodingWithSigHashType(const valtype &vchSig,
                                     validSig, flags, &err));
 
         // If we have strict encoding, we prevent the use of undefined flags.
-        std::array<SigHashType, 2> undefSigHashes{
-            {SigHashType(sigHash.getRawSigHashType() | 0x20),
+        std::array<SigHashType, 3> undefSigHashes{
+            {SigHashType(sigHash.getRawSigHashType() | 0x10),
```

Comment:
Where's this guy coming from? Just to ensure we don't have one of the three reserved hits? Might want to add them all then?

Reply:
Makes sense, will fix.